packages/test-commons/bootstrap.js

```diff
@@ -2,9 +2,13 @@
  * Copyright 2017 Palantir Technologies, Inc. All rights reserved.
  */
 
-import "./polyfill";
+require("./polyfill");
```

Comment:
why is it necessary to switch to commonjs for this test code? should the node version that runs it support es6 modules?

Reply:
necessary because isotest uses this now and that suite explicitly runs in a node environment (for server-side rendering)